modules/Q02_openai_question.sh

```diff
@@ -46,80 +46,80 @@ Q02_openai_question() {
 # looks through the modules and finds chatgpt questions inside the csv
 ask_chatgpt() {
   local GPT_FILE_DIR_="${LOG_PATH_MODULE}""/gpt_files/"
+  local GPT_PRIO_=3
+  # default vars
+  local GPT_QUESTION_="" 
+  local CHATGPT_CODE_=""
+  local GPT_RESPONSE_=""
+  local GPT_RESPONSE_CLEANED_=""
+  local GPT_TOKENS_=0
+  local HTTP_CODE_=200
+  local ORIGIN_MODULE_=""
   print_output "[*] Checking scripts with ChatGPT that have priority ${MINIMUM_GPT_PRIO} or lower" "no_log"
-    local GPT_PRIO_=3
-    # default vars
-    local GPT_QUESTION_="" 
-    local CHATGPT_CODE_=""
-    local GPT_RESPONSE_=""
-    local GPT_RESPONSE_CLEANED_=""
-    local GPT_TOKENS_=0
-    local HTTP_CODE_=200
-    local ORIGIN_MODULE_=""
-
-    while IFS=";" read -r COL1_ COL2_ COL3_ COL4_ COL5_ COL6_ COL7_; do
-      SCRIPT_PATH_TMP_="${COL1_}"
-      GPT_ANCHOR_="${COL2_}"
-      GPT_PRIO_="${COL3_//GPT-Prio-/}"
-      GPT_QUESTION_="${COL4_}"
-      GPT_OUTPUT_FILE_="${COL5_}"
-      GPT_TOKENS_="${COL6_//cost\=/}"
-      GPT_RESPONSE_="${COL7_}"
-      GPT_INPUT_FILE_="$(basename "${SCRIPT_PATH_TMP_}")"
-      
-      print_output "[*]trying to check inside ${LOG_DIR}/firmware" "no_log"
-      SCRIPT_PATH_TMP_="$(find "${LOG_DIR}/firmware" -wholename "*${SCRIPT_PATH_TMP_}")"
-      cp "${GPT_INPUT_FILE_}" "${GPT_FILE_DIR_}/${GPT_INPUT_FILE_}.log"
-      print_output "[*]trying to check ${SCRIPT_PATH_TMP_} with Question ${GPT_QUESTION_} " "no_log"
-      print_output "[*]Prio is ${GPT_PRIO_}"  "no_log"
+  mkdir "${GPT_FILE_DIR_}"
+  while IFS=";" read -r COL1_ COL2_ COL3_ COL4_ COL5_ COL6_ COL7_; do
+    SCRIPT_PATH_TMP_="${COL1_}"
+    GPT_ANCHOR_="${COL2_}"
+    GPT_PRIO_="${COL3_//GPT-Prio-/}"
+    GPT_QUESTION_="${COL4_}"
+    GPT_OUTPUT_FILE_="${COL5_}"
+    GPT_TOKENS_="${COL6_//cost\=/}"
+    GPT_RESPONSE_="${COL7_}"
+    GPT_INPUT_FILE_="$(basename "${SCRIPT_PATH_TMP_}")"
+    
+    print_output "[*]trying to check inside ${LOG_DIR}/firmware" "no_log"
```

Comment:
Please include a space between [*] and your output text. Additionally please start with capital Letters.
I would suggest something like the following:
```
    print_output "[*] Trying to check inside ${ORANGE}${LOG_DIR}/firmware${NC}" "no_log"
```